subiquity/common/apidef.py

```diff
@@ -375,6 +376,13 @@ def POST(data: Payload[IdentityData]): ...
         class validate_username:
             def GET(username: str) -> UsernameValidation: ...
 
+    class ssh:
```

Comment:
oh, you forgot to remove the initial definition of the ssh endpoint i think?

Reply:
oh yeah, good catch!